package.json

```
@@ -10,7 +10,8 @@
   "workspaces": [
     "common",
     "server",
-    "client"
+    "client",
+    "ott-vis"
```

it needs to be the full path from the root of the repo
```suggestion
    "packages/ott-vis"
```